images/test-runner/Makefile

```
@@ -39,7 +37,7 @@ build: ensure-buildx
 		--progress=$(PROGRESS) \
 		--pull \
 		--build-arg BASE_IMAGE=$(NGINX_BASE_IMAGE) \
-		--build-arg GOLANG_VERSION=1.15.6 \
+		--build-arg GOLANG_VERSION=1.16.3 \
```

```suggestion
		--build-arg GOLANG_VERSION=1.16.5 \
```

the latest golang version is 1.16.5